WaterLog_Backend/Migrations/20190226074431_TankTables2.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace WaterLog_Backend.Migrations
+{
+    public partial class TankTables2 : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<DateTime>(
+                name: "TimeStamp",
+                table: "TankReadings",
+                nullable: false,
+                defaultValue: new DateTime(1, 1, 1, 0, 0, 0, 0, DateTimeKind.Unspecified));
```

Comment:
This looks weird, why not use current timestamp or something like that and also why not specify the datetimekind

Reply:
the current time stamp is used. We need all the readings with different timestamps for data collection and graph population